### PR TITLE
adding README.MD

### PR DESCRIPTION
**Pull request by daglew**

### CODE REVIEW

File: environment/README.MD [hunk 1]
```diff
@@ -1 +1,128 @@
...
+* [Clone repository](#clone-repository-using-git-bash)
+* [Create a new branch](#create-a-new-branch)
+* [Create prepare_environment file](#create-prepare_environment-file)
+
+
+
+## Download python
+
+```
+1. Go to https://www.python.org/downloads/
```
Comment (babelag) [inline environment/README.MD]: change this as a link https://www.python.org/downloads/

File: environment/README.MD [hunk 2]
```diff
@@ -1 +1,128 @@
...
+2. Download python 3.10.4
+3. Install it.
+4. Click windows button+r on your keyboard.
+5. Please provide "cmd" command
```
Comment (babelag) [inline environment/README.MD]: delete Please

File: environment/README.MD [hunk 3]
```diff
@@ -1 +1,128 @@
...
+6. Please use command "python3 --version" in cmd
```
Comment (babelag) [inline environment/README.MD]: Delete please

File: environment/README.MD [hunk 4]
```diff
@@ -1 +1,128 @@
...
+7. Check version of installed python.
+```
+
+## Download pycharm
+
+```
+1. Go to PyCharm
```
Comment (babelag) [inline environment/README.MD]: where link to Pycharm download, add as a link

File: environment/README.MD [hunk 5]
```diff
@@ -1 +1,128 @@
...
+2. Take community version.
+3. Install it
+```
+
+## Download git
+
+```
+1. Go to Git - Downloading Package
```
Comment (babelag) [inline environment/README.MD]: where link to Git download, add as a link

File: environment/README.MD [hunk 6]
```diff
@@ -1 +1,128 @@
...
+7. Click "code" button
+8. Copy HTTPS link.
+9. Back to Git Bash
+10. Clone repository to your local machine use command "git clone <copied link>"
+11. Check if folder "learning" is visible in folder "repositories"
+``` 
+## Create a new branch
+
+```
+1.Open Pycharm
```
Comment (babelag) [inline environment/README.MD]: add space after 1.

File: environment/README.MD [hunk 7]
```diff
@@ -1 +1,128 @@
...
+9. Check on what branch are you
+10. Use command "git checkout task_1-first branch"
+11. Check on what branch are you
+12. Go to develop branch
+13. Delete the branch "git branch -d <task_1-first branch>
+``` 
+## Create prepare_environment file
+
+```
+1. Go to the Notepad ++
```
Comment (babelag) [inline environment/README.MD:100]: where link to Notepadd++ download, please add it

**Review by babelag — CHANGES_REQUESTED** [review]

File: environment/README.MD (hunk 1 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 2 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 3 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 4 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 5 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 6 above)
Comment (daglew) [reply environment/README.MD]: Done

File: environment/README.MD (hunk 7 above)
Comment (daglew) [reply environment/README.MD:100]: Done

File: README.md [hunk 8]
```diff
@@ -1,3 +1,42 @@
-# learning
+# Learning
 It is my first repository, there will be all information about my learning
 
+## Table of contents
+* [Links](#links)
```
Comment (babelag) [inline README.md:5]: Where is a link to Linux Commands ?

File: README.md [hunk 9]
```diff
@@ -1,3 +1,42 @@
...
+* [Git Commands](#git-commands)
+
+## Links
+Readme:
+[python](https://www.python.org/downloads/)
```
Comment (babelag) [inline README.md]: LINKS TO README helpers pages not to python, pycharm, git, notepad 👌

**Review by babelag — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -1,3 +1,42 @@
...
+CTRL + C zatrzymuje proces
+clear- czyści konsole
+CTRL + X zamykam nano
+nano edytuje plik
+```
+
+## Git Commands
+```
+git clone <repozytorium> - klonowanie repo
+
```
Comment (babelag) [inline README.md]: please remove this line

**Review by babelag — CHANGES_REQUESTED** [review]

File: README.md [hunk 11]
```diff
@@ -1,3 +1,42 @@
...
 It is my first repository, there will be all information about my learning
 
+## Table of contents
+* [Links](#links)
+* [Git Commands](#git-commands)
+
+## Links
+Readme:
+[python](https://www.python.org/downloads/)
+[pycharm](https://www.jetbrains.com/pycharm/)
```
Comment (babelag) [inline README.md]: please use "* " as a first sign of line

**Review by babelag — CHANGES_REQUESTED** [review]

File: environment/README.MD [hunk 12]
```diff
@@ -1 +1,127 @@
...
+Go to [git](https://git-scm.com/download/win)
+```
+1. Download 32-bits version.
+2. Install it.
+```
+
+## Install Notepad++
+Go to [notepad++](https://notepad-plus-plus.org/downloads/v8.4.1/)
+```
+1. Go to Download Notepad++ v8.4.1 | Notepad++.
```
Comment (babelag) [inline environment/README.MD]: please remove " | Notepad++."

**Review by babelag — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (daglew) [reply README.md:5]: Done

File: README.md (hunk 9 above)
Comment (daglew) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (daglew) [reply README.md:5]: Done

File: README.md (hunk 10 above)
Comment (daglew) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (daglew) [reply README.md]: Done

File: environment/README.MD (hunk 12 above)
Comment (daglew) [reply environment/README.MD]: Done

**Pull request merged**